nomad/structs/structs_test.go

```diff
@@ -5420,6 +5420,7 @@ func TestTaskEventPopulate(t *testing.T) {
 		{NewTaskEvent(TaskSetupFailure).SetSetupError(fmt.Errorf("task failed setup")), "task failed setup"},
 		{NewTaskEvent(TaskDriverFailure), "Failed to start task"},
 		{NewTaskEvent(TaskDownloadingArtifacts), "Client is downloading artifacts"},
+		{NewTaskEvent(TaskReDownloadingArtifacts), "Client is re-downloading artifacts since `always_fetch` is true"},
```

Comment:
If the above suggestion is accepted this needs updating

```suggestion
		{NewTaskEvent(TaskReDownloadingArtifacts), "Client is re-downloading artifacts since always_fetch=true"},
```